### ISSUE TITLE
KeyError: u'0500000US72013' while running example choropleth_counties.py

### ISSUE THREAD

**Issue by raghavchalapathy** [python-visualization/folium#402]
Hi 
Thanks for this wonderfull tool 

Could you please give me some pointers as to why am I getting the error below 

I am running  "choropleth_counties.py" present in the examples directory

```shell
/Users/raghav/PycharmProjects/Visualization/interactive-Maps/folium/examples/choropleth_counties.py:48: FutureWarning: Map.create_map is deprecated. Use Map.save instead
  map_1.create_map(path='map_1.html')
Traceback (most recent call last):
  File "/Users/raghav/PycharmProjects/Visualization/interactive-Maps/folium/examples/choropleth_counties.py", line 48, in <module>
    map_1.create_map(path='map_1.html')
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/folium.py", line 127, in create_map
    self.save(path)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/element.py", line 153, in save
    html = root.render(**kwargs)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/element.py", line 359, in render
    child.render(**kwargs)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/element.py", line 667, in render
    element.render(**kwargs)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/features.py", line 480, in render
    super(TopoJson, self).render(**kwargs)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/element.py", line 663, in render
    figure.script.add_children(Element(script(self, kwargs)),
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/jinja2/runtime.py", line 436, in __call__
    return self._func(*arguments)
  File "<template>", line 15, in macro
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/jinja2/runtime.py", line 196, in call
    return __obj(*args, **kwargs)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/features.py", line 475, in style_data
    feature.setdefault('properties', {}).setdefault('style', {}).update(self.style_function(feature))  # noqa
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/folium.py", line 679, in style_function
    "fillColor": color_scale_fun(x)
  File "/Users/raghav/anaconda/lib/python2.7/site-packages/folium/folium.py", line 668, in color_scale_fun
    u <= color_data[get_by_key(x, key_on)]])]
KeyError: u'0500000US72013'
```

**Comment by KonniamChan** [comment]
I ran into the same error as well. Would appreciate any pointers on the issue.

**Comment by ocefpaf** [comment]
`choropleth_counties.py` uses the old syntax (`geo_json`) instead of the new one (`choropleth`). Still, it seems to be broken for TopoJSON.

I don't have time to look at this right now, so I recommend as a work around to convert the TopoJSON to GeoJSON and use that instead.

@BibMartin can you see any obvious  problems with an easy fix?

**Comment by rohanp** [comment]
I am also getting the same error

**Comment by BibMartin** [comment]
This is mainly due to data : the join between counties data and counties topojson does not fit perfectly.
See changes in #456.

Note that with new choropleth default behavior, the output map has few interest (almost every county is in yellow) ; it would worth providing a `threshold_scale` in the example.

**Comment by ocefpaf** [comment]
Fixed in #456
